src/herder/HerderImpl.cpp

```diff
@@ -746,19 +746,27 @@ HerderImpl::sendSCPStateToPeer(uint32 ledgerSeq, Peer::pointer peer)
 
     bool delayCheckpoint = false;
     auto checkpoint = getMostRecentCheckpointSeq();
-    auto highestSlot = getSCP().getHighestKnownSlotIndex();
+    auto firstSequentialLedgerSeq =
```

Comment:
Herder tracking can be less than MAX_SLOTS_TO_REMEMBER, so we need to handle that (by setting firstSequentialLedgerSeq to LedgerManager::GENESIS_LEDGER_SEQ)